XenModel/InvisibleMessages.ja.resx

```diff
@@ -178,7 +178,7 @@
     <value>xsupdate</value>
   </data>
   <data name="ISO_UPDATE" xml:space="preserve">
-    <value>ISO</value>
+    <value>iso</value>

```

Comment:
Nit: It might be worth removing this as a translatable string and having it as a constant in code instead.